airflow/providers/__init__.py

```diff
@@ -14,3 +14,16 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+from airflow.config_templates.airflow_local_settings import DEFAULT_LOGGING_CONFIG, LOG_LEVEL
+from airflow.configuration import conf
+
+PROVIDERS_LOGGING_SETTINGS: str = conf.get('providers_google', 'VERBOSE_LOGGING')
+if PROVIDERS_LOGGING_SETTINGS == "true":
+    for a in ["google_auth_httplib2", "httplib2", "googleapiclient"]:
```

Comment:
Let's change the name to something different from `a` :D

Reply:
Yeah. I am rewriting it a bit!